chromadb/test/distributed/test_sanity.py

```diff
@@ -98,61 +84,14 @@ def test_add_include_all_with_compaction_delay(client: ClientAPI) -> None:
     print("Generated data with seed ", seed)
 
     # Query the collection with a random query
-    results = collection.query(
-        query_embeddings=[random_query_1, random_query_2],  # type: ignore
-        n_results=10,
-        include=["metadatas", "documents", "distances", "embeddings"],  # type: ignore[list-item]
+    invariants.ann_accuracy(
+        collection,
+        {
+            "ids": ids,
+            "embeddings": embeddings,  # type: ignore[typeddict-item]
+            "metadatas": None,
+            "documents": documents,
+        },
+        10,
+        query_embeddings=[random_query_1, random_query_2],
     )
-
-    ids_and_embeddings = list(zip(ids, embeddings))
-
-    def validate(results: QueryResult, query: List[float], result_index: int) -> None:
-        # Check that the distances are correct in l2
-        gt_ids_distances_embeddings = [
-            (id, sum((a - b) ** 2 for a, b in zip(embedding, query)), embedding)
-            for id, embedding in ids_and_embeddings
-        ]
-        gt_ids_distances_embeddings.sort(key=lambda x: x[1])
-        retrieved_distances = results["distances"][result_index]  # type: ignore
-
-        # Check that the query results are sorted by distance
-        for i in range(1, len(retrieved_distances)):
-            assert retrieved_distances[i - 1] <= retrieved_distances[i]
-
-        for i in range(len(retrieved_distances)):
-            assert abs(gt_ids_distances_embeddings[i][1] - retrieved_distances[i]) < EPS
-
-        # Check that the ids are correct
-        retrieved_ids = results["ids"][result_index]
-        for i in range(len(retrieved_ids)):
-            assert retrieved_ids[i] == gt_ids_distances_embeddings[i][0]
-
-        # Check that the documents are correct
-        if "documents" in results and results["documents"] is not None:
-            retrieved_documents = results["documents"][result_index]
-            for i in range(len(retrieved_documents)):
-                assert (
-                    retrieved_documents[i]
-                    == f"document_{gt_ids_distances_embeddings[i][0]}"
-                )
-        else:
-            assert False
-
-        # Check that the embeddings are correct
-        if "embeddings" in results and results["embeddings"] is not None:
-            retrieved_embeddings = results["embeddings"][result_index]
-            for i in range(len(retrieved_embeddings)):
-                # eps compare the embeddings
-                for j in range(3):
-                    assert (
-                        abs(
-                            retrieved_embeddings[i][j]
-                            - gt_ids_distances_embeddings[i][2][j]
-                        )
-                        < EPS
-                    )
-        else:
-            assert False
-
-    validate(results, random_query_1, 0)
-    validate(results, random_query_2, 1)
```

Comment:
this only checked the first 3 results, so the updated test is strictly better and performs the same checks within `ann_accuracy`:

- correct distances
- sorted by distance
- IDs match expected
- documents match
- embeddings match